README.md

```diff
@@ -1 +1,162 @@
-# Math-Magicians
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Math Magicians setup</b></h3>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+**Math magicians** Math magicians is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+-Make simple calculations.
+-Read a random math-related quote.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+  <li><a href="#">HTML</a></li>
+  <li><a href="#">CSS</a></li>
+    <li><a href="#">React vite</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="NA">NA</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Make Math calculation**
+- **Read a random math-related quote**
+<p align="right">
+(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- comming soon
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** As your project is not deployed yet, I think it would be better if you remove the `Live Demo` from and add it it later with the working link or **you can leave it and deploy it now**. This will make your documentation to be professional as leaving empty sections in your documentation is not professional.